archive/j/javascript/oddNumbers.js

```diff
@@ -0,0 +1,15 @@
+function printOdd(num) {
+  for (let i=1; i<num; i++) {
+    if(i % 2 !== 0) {
+      console.log(i);
+    }
+  }
+}
+
+function printEven(num) {
+  for (let i=1; i<num; i++) {
+    if(i % 2 === 0) {
+      console.log(i);
+    }
+  }
+}
```

Comment:
It would be more concise to do something like this
```javascript
function printEvenOdd(num) {
    // Assume the number is Even
    const result = "Even";
    // Test to see if it is Odd
    if ( num % 2 !== 0 ) {
        // If it is Odd change the result
        result = "Odd";
    }
    // Print the result to the console
    console.log(result)
}
```